src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -62,8 +73,8 @@ struct cbuf_maps {
 };
 
 struct cbuf_info {
-	u32_t cbid;
-	int size;
+	int cbid;
```

Comment:
While we agree in the cbuf_info struct this field should be unsigned, we think it would take a while to fix this throughout the code, while also minimizing casts. For now, Yuxin will attempt to merge as-is, and I will try to fix on my end. This might also be a big enough issue to discuss in person.